routes/bot.js

```diff
@@ -61,4 +64,27 @@ router.post("/ranking", urlencodedParser, async (req, res) => {
   res.json(response);
 });
 
+router.post("/feedback", urlencodedParser, async (req, res) => {
+  let user = {};
+  try {
+    user = await userController.find(req.body.user_id);
+
+    const url = `https://slack.com/api/chat.postEphemeral?token=${
+      process.env.SLACK_TOKEN
+    }&channel=${config.channels.valid_channels[0]}&text=${encodeURIComponent(
+      `Tio, ${user.name} mandou um super feedback, saca só: _${req.body.text}_`
+    )}&user=UCJA2A8Q5&pretty=1`;
```

Comment:
`user=UCJA2A8Q5` This id changes in staging, right? If so, it would have to be an environment variable.